content/posts/newbie.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
may I ask whom this post targets?

I think I got your idea but I feel like there are just too many fancy words(esoteric aspects, architecture astronauts) that need their own explanation.

One problem with unfamiliar words ( or words with deep meaning ) is that it takes time to understand them, hence it breaks the flow of reading the blog.

surely you read this but I think it's a good reminder: http://www.paulgraham.com/simply.html